pkg/reconciler/autoscaling/hpa/resources/hpa.go

```diff
@@ -77,7 +77,7 @@ func MakeHPA(pa *v1alpha1.PodAutoscaler, config *autoscalerconfig.Config) *autos
 			Type: autoscalingv2beta1.ObjectMetricSourceType,
 			Object: &autoscalingv2beta1.ObjectMetricSource{
 				Target: autoscalingv2beta1.CrossVersionObjectReference{
-					APIVersion: servingv1alpha1.SchemeGroupVersion.String(),
+					APIVersion: servingv1.SchemeGroupVersion.String(),
```

Comment:
It'd be nice to drop the version here

Reply:
I frankly dunno how these CrossVersionObjectReferences work. Does K8s HPA support dropping it? I suppose we could do that in 0.14 then?

Reply:
My guess is the version is needed since metrics could change (or be introduced) in newer versions